packages/mongodb-memory-server-core/src/util/MongoBinary.ts

```diff
@@ -128,17 +146,16 @@ export default class MongoBinary {
       checkMD5: envToBool(resolveConfig('MD5_CHECK') ?? ''),
     };
 
+    /** Provided Options combined with the Default Options */
     const options = { ...defaultOptions, ...opts };
-    log(`MongoBinary options: ${JSON.stringify(options)}`);
-
-    const { version, systemBinary } = options;
```

Comment:
removed these because they were not changed after this and wasn't shortening much

Reply:
agreed